timbermill-client/src/main/java/com/datorama/timbermill/ElasticsearchClient.java

```diff
@@ -198,6 +234,37 @@ void index(Map<String, Task> tasksMap) {
         }
     }
 
+    void rolloverIndex(String timbermillAlias) {
+        RolloverRequest rolloverRequest = new RolloverRequest(timbermillAlias, null);
```

Comment:
if it's the same request every time we can generate it once and store as a final member

Reply:
It' changes per env